helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -235,7 +235,35 @@ receivers:
               from: log
               to: $$
   {{- end }}
-
+  # https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/journaldreceiver
+  {{- if .Values.logsCollection.journaldLogs.enabled }}
```

Comment:
let's make it into `    {{- if and (eq .Values.logsEngine "otel") .Values.logsCollection.journaldLogs.enabled }}`

Reply:
actually we also need this `{{- if (eq (include "splunk-otel-collector.logsEnabled" .) "true") }}`

Reply:
Thanks @rockb1017.